app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class BenefitsIntakeStatusJob
+  include Sidekiq::Job
+
+  def perform
+    pending_form_submission_ids = FormSubmission
+                                  .joins(:form_submission_attempts)
+                                  .where('form_submission_attempt.aasm_state = pending')
+                                  .map(&:benefits_intake_uuid)
+    response = BenefitsIntakeService::Service.get_bulk_status_of_uploads(pending_form_submission_ids)
+    handle_response(response) if response.status == 200
```

Comment:
will this ever actually run?  seems like `get_bulk_status_of_uploads` will raise an exception if we get a failure response 🤔

Reply:
You're totally right; this is redundant. Thanks! 😄